repos/system_upgrade/el7toel8/actors/sanebackendsmigrate/libraries/library.py

```diff
@@ -0,0 +1,324 @@
+    '#fi-7180',
+    'usb 0x04c5 0x132c',
+    '#fi-7280',
+    'usb 0x04c5 0x132d',
+    '#fi-7160',
+    'usb 0x04c5 0x132e',
+    '#fi-7260',
+    'usb 0x04c5 0x132f',
+    '#ScanSnap iX500EE',
+    'usb 0x04c5 0x13f3',
+    '#ScanSnap iX100',
+    'usb 0x04c5 0x13f4',
+    '#ScanPartner SP25',
+    'usb 0x04c5 0x1409',
+    '#ScanPartner SP30',
+    'usb 0x04c5 0x140a',
+    '#ScanPartner SP30F',
+    'usb 0x04c5 0x140c',
+    '#fi-6140ZLA',
+    'usb 0x04c5 0x145f',
+    '#fi-6240ZLA',
+    'usb 0x04c5 0x1460',
+    '#fi-6130ZLA',
+    'usb 0x04c5 0x1461',
+    '#fi-6230ZLA',
+    'usb 0x04c5 0x1462',
+    '#fi-6125ZLA',
+    'usb 0x04c5 0x1463',
+    '#fi-6225ZLA',
+    'usb 0x04c5 0x1464',
+    '#fi-6135ZLA',
+    'usb 0x04c5 0x146b',
+    '#fi-6235ZLA',
+    'usb 0x04c5 0x146c',
+    '#fi-6120ZLA',
+    'usb 0x04c5 0x146d',
+    '#fi-6220ZLA',
+    'usb 0x04c5 0x146e',
+    '#N7100',
+    'usb 0x04c5 0x146f',
+    '#fi-6400',
+    'usb 0x04c5 0x14ac',
+    '#fi-7480',
+    'usb 0x04c5 0x14b8',
+    '#fi-6420',
+    'usb 0x04c5 0x14bd',
+    '#fi-7460',
+    'usb 0x04c5 0x14be',
+    '#fi-7140',
+    'usb 0x04c5 0x14df',
+    '#fi-7240',
+    'usb 0x04c5 0x14e0',
+    '#fi-7135',
+    'usb 0x04c5 0x14e1',
+    '#fi-7235',
+    'usb 0x04c5 0x14e2',
+    '#fi-7130',
+    'usb 0x04c5 0x14e3',
+    '#fi-7230',
+    'usb 0x04c5 0x14e4',
+    '#fi-7125',
+    'usb 0x04c5 0x14e5',
+    '#fi-7225',
+    'usb 0x04c5 0x14e6',
+    '#fi-7120',
+    'usb 0x04c5 0x14e7',
+    '#fi-7220',
+    'usb 0x04c5 0x14e8',
+    '#fi-400F',
+    'usb 0x04c5 0x151e',
+    '#fi-7030',
+    'usb 0x04c5 0x151f',
+    '#fi-7700',
+    'usb 0x04c5 0x1520',
+    '#fi-7600',
+    'usb 0x04c5 0x1521',
+    '#fi-7700S',
+    'usb 0x04c5 0x1522'
+]
+
+CANON = [
+    '# Canon LiDE 80',
+    'usb 0x04a9 0x2214',
+    '# Canon LiDE 120',
+    'usb 0x04a9 0x190e',
+    '# Canon LiDE 220',
+    'usb 0x04a9 0x190f'
+]
+
+XEROX_MFP = [
+    '#Samsung X4300 Series',
+    'usb 0x04e8 0x3324',
+    '#Samsung K4350 Series',
+    'usb 0x04e8 0x3325',
+    '#Samsung X7600 Series',
+    'usb 0x04e8 0x3326',
+    '#Samsung K7600 Series',
+    'usb 0x04e8 0x3327',
+    '#Samsung K703 Series',
+    'usb 0x04e8 0x3331',
+    '#Samsung X703 Series',
+    'usb 0x04e8 0x3332',
+    '#Samsung M458x Series',
+    'usb 0x04e8 0x346f',
+    '#Samsung M4370 5370 Series',
+    'usb 0x04e8 0x3471',
+    '#Samsung X401 Series',
+    'usb 0x04e8 0x3477',
+    '#Samsung K401 Series',
+    'usb 0x04e8 0x3478',
+    '#Samsung K3250 Series',
+    'usb 0x04e8 0x3481',
+    '#Samsung X3220 Series',
+    'usb 0x04e8 0x3482'
+]
+
+"""
+Dictionary of configuration files which changes in 1.0.27
+"""
+
+NEW_QUIRKS = {
+    '/etc/sane.d/canon_dr.conf': CANON_DR,
+    '/etc/sane.d/cardscan.conf': CARDSCAN,
+    '/etc/sane.d/dll.conf': DLL,
+    '/etc/sane.d/epjitsu.conf': EPJITSU,
+    '/etc/sane.d/fujitsu.conf': FUJITSU,
+    '/etc/sane.d/canon.conf': CANON,
+    '/etc/sane.d/xerox_mfp.conf': XEROX_MFP
+}
```

Comment:
```python
NEW_QUIRKS = {
    '/etc/sane.d/canon_dr.conf': CANON_DR,
    '/etc/sane.d/cardscan.conf': CARDSCAN,
    '/etc/sane.d/dll.conf': DLL,
    '/etc/sane.d/epjitsu.conf': EPJITSU,
    '/etc/sane.d/fujitsu.conf': FUJITSU,
    '/etc/sane.d/canon.conf': CANON,
    '/etc/sane.d/xerox_mfp.conf': XEROX_MFP
}
"""
Dictionary of configuration files which changed in 1.0.27
"""

Reply:
You're right, thank you for checking - it will be fixed in the newest commit.